modules/RTC/TPCUtils.js

```diff
@@ -345,15 +422,22 @@ export class TPCUtils {
      * type and other considerations associated with screenshare.
      *
      * @param {JitsiLocalTrack} localVideoTrack The local video track.
+     * @param {CodecMimeType} codec - The codec currently in use.
      * @returns {Array<number>}
      */
-    calculateEncodingsBitrates(localVideoTrack) {
+    calculateEncodingsBitrates(localVideoTrack, codec) {
+        // SVC mode for VP9 and AV1 codecs.
+        if (this._isRunningInSvcMode(codec)) {
+            return [ this.codecSettings[codec].maxBitratesVideo.high, 0, 0 ];
```

Comment:
Can this be called for a SS track? Shouldn't it be `ssHigh` in that case?

Reply:
The bitrate for the first encoding is updated here - https://github.com/jitsi/lib-jitsi-meet/pull/2350/files#diff-6cd0aeb8cb1acbd0118edb9d36b059e32f1984192a2c012bf5d6db68f02ab71aR726
This line is here to ensure that the other 2 encodings have a max bitrate of 0 so that they are disabled all the time.

Reply:
I don't think that is obvious at all from an API prespective... is there a way to unify the logic for all encodings?

Reply:
Ok sure.